src/components/interaction-designer/block-editors/BlockEditor.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="block-editor">
+  <div class="block-editor" :style="{top: topPosition}">
```

Comment:
the major implementation is in this file:
- updated the css
- introduced a dynamic top value